pkg/cache/cache.go

```diff
@@ -148,6 +148,20 @@ func (c *Cache) newClusterQueue(cq *kueue.ClusterQueue) (*ClusterQueue, error) {
 	return cqImpl, nil
 }
 
+func (c *Cache) AdmittedWorkloadsLocalQueue(localQueue *kueue.LocalQueue) (int32, error) {
+	c.Lock()
+	defer c.Unlock()
+	cq, ok := c.clusterQueues[string(localQueue.Spec.ClusterQueue)]
+	if !ok {
+		return 0, nil
+	}
+	localKey := queueKey(localQueue)
```

Comment:
```suggestion
	qKey := queueKey(localQueue)
```

or `localQueueKey` if you want to be more explicit.